pmd-vm/src/main/java/net/sourceforge/pmd/lang/vm/ast/AbstractVmNode.java

```diff
@@ -155,6 +155,7 @@ public String toString(final String prefix) {
      */
     @Deprecated
     public void dump(final String prefix, final boolean recurse, final Writer writer) {
+        @SuppressWarnings("PMD.CloseResource")
```

Comment:
Same potential FP: The stream is provided from outside (here as a method parameter), so we should not be responsible here to close it, should we?